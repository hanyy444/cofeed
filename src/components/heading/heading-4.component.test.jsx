import { render, screen } from '@testing-library/react'
import HeadingFour from './heading-4.component'

describe('Test heading-4 component', () => { 
    it('Should render without crashing', () => {
        expect.assertions(2)
        render(<HeadingFour>test</HeadingFour>)
        const headingFour = screen.getByTestId('heading-4')
        expect(headingFour).toBeInTheDocument()
        expect(headingFour).toMatchSnapshot()
    })
})