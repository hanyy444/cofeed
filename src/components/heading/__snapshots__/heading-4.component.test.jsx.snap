// Vitest Snapshot v1

exports[`Test heading-4 component > Should render without crashing 1`] = `
<h4
  class="heading-4"
  data-testid="heading-4"
>
  test
</h4>
`;
